app/_locales/en/messages.json

```diff
@@ -2754,13 +2754,13 @@
     "message": "View More"
   },
   "viewOnCustomBlockExplorer": {
-    "message": "View at $1"
+    "message": "View $1 at $2"
```

Comment:
[REQ] When we add variables to messages our current best practices are to include a description field that explains what those variables will be. There are examples in the messages.json file. I also realize that the original didn't have this but its a good time to add it now that we are extending it